identity-apps-core/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalConstants.java

```diff
@@ -53,9 +53,9 @@ private AppPortalConstants() {
      */
     public enum AppPortal {
 
-        MY_ACCOUNT("My Account", "This is the my account application.", "MY_ACCOUNT", "/myaccount",
+        MY_ACCOUNT("My Account", "This is the my account application.", "MY_ACCOUNT", "/(.*)",
                 "/myaccount/"),
-        CONSOLE("Console", "This is the console application.", "CONSOLE", "/console",
+        CONSOLE("Console", "This is the console application.", "CONSOLE", "/(.*)",
```

Comment:
@brionmario Should we derive this using https://github.com/wso2/identity-apps/blob/master/apps/console/src/public/deployment.config.json#L15 without a wildcard